docs/plugin_fundamentals/import_automation.md

```diff
@@ -371,10 +353,7 @@ public class ImportRunner : IImportRunner
       Path = partPath + PathElement.Char(characteristic)
     };
 
-    await restClient.CreateCharacteristics(
-        [knownCharacteristic],
-        cancellationToken: cancellationToken)
-      .ConfigureAwait(false);
+    await restClient.CreateCharacteristics([knownCharacteristic], cancellationToken: cancellationToken).ConfigureAwait(false);
```

Comment:
We have to see if the line at the end is too long for the full page width, that's why the line breaks were inserted.

Reply:
Ah fair point, I'll have a look at this.

Reply:
All lines should fit now.